### PR TITLE
Fixes #685 : fix default values of reader settings

### LINKED ISSUE #685: app first start (default settings) => reader view options are not initialized correctly (store sync problem?)
ThoriumDefaults1
ThoriumDefaults2

I noticed that changing the display setting from scroll to paginated results in the font size slider being set to the correct default value (see screenshots below), but the colour theme radio buttons are not updated (the tick is missing) ... in fact, nothing happens when I click on the default neutral option, only when I select the other ones.

ThoriumDefaults3
ThoriumDefaults4

### PR DESCRIPTION
**Pull request by Scarsniik**

### CODE REVIEW

**Comment by danielweck** [comment]
Hello, would you mind removing the changes to `package-lock.json`? Thanks! :)

File: src/renderer/components/reader/ReaderOptions.tsx [hunk 1]
```diff
@@ -100,58 +98,57 @@ export class ReaderOptions extends React.Component<Props> {
...
+                    </label>
                 </div>
-            </>
+            </div>
         );
     }
 
     private textContent() {
         const {__, settings} = this.props;
+        console.log(this.props.indexes.fontSize);
```
Comment (danielweck) [inline src/renderer/components/reader/ReaderOptions.tsx]: Please use `debug()` or remove the `console.log()` statement.

File: src/renderer/components/reader/ReaderOptions.tsx [hunk 2]
```diff
@@ -245,7 +242,7 @@ export class ReaderOptions extends React.Component<Props> {
                             name="alignment"
                             type="radio"
                             onChange={(e) => this.props.handleSettingChange(e, "align", "left")}
-                            checked={this.props.settings.align === "left"}
+                            checked={settings.align === "left"}
```
Comment (danielweck) [inline src/renderer/components/reader/ReaderOptions.tsx]: This data structure is actually strongly-typed, via `ReaderConfig` (for example, "left" is recognized as a valid enum value for `align`). So, whilst you are editing code in `ReaderOptions.tsx`, could you please change `settings: any;` to `settings: ReaderConfig;` in `interface Props extends TranslatorProps` at the top of the file? Thank you! (`import { ReaderConfig } from "readium-desktop/common/models/reader";`)

PS: Pierre is working in separate PRs for further removal of `any` and better typing across the board.

File: src/renderer/components/reader/ReaderOptions.tsx [hunk 3]
```diff
@@ -261,7 +258,7 @@ export class ReaderOptions extends React.Component<Props> {
                             name="alignment"
                             type="radio"
                             onChange={(e) => this.props.handleSettingChange(e, "align", "justify")}
-                            checked={this.props.settings.align === "justify"}
+                            checked={settings.align === "justify"}
```
Comment (danielweck) [inline src/renderer/components/reader/ReaderOptions.tsx]: Replace `"justify"` with `textAlignEnum.justify`
(`import { textAlignEnum } from "@r2-navigator-js/electron/common/readium-css-settings";`)

File: src/renderer/components/reader/ReaderOptions.tsx (hunk 2 above)
Comment (danielweck) [inline src/renderer/components/reader/ReaderOptions.tsx]: Replace `"left"` with `textAlignEnum.left`
(`import { textAlignEnum } from "@r2-navigator-js/electron/common/readium-css-settings";`)

File: src/renderer/components/reader/ReaderOptions.tsx [hunk 4]
```diff
@@ -100,58 +98,57 @@ export class ReaderOptions extends React.Component<Props> {
...
+                    <input
                         id={"radio-" + themeType.Without}
                         type="radio"
                         name="theme"
                         onChange={() => this.handleChooseTheme(themeType.Without)}
-                        {...(!this.props.settings.sepia && !this.props.settings.night
-                        && {checked: true})}
-                        />
-                        <label htmlFor={"radio-" + themeType.Without}>
-                            {this.props.settings.predefined && <SVG svg={DoneIcon} ariaHidden/>}
```
Comment (danielweck) [inline src/renderer/components/reader/ReaderOptions.tsx]: I think the line of code `values.predefined = neutral;` must be removed too, in the `handleChooseTheme()` function. Is that correct?

Comment (danielweck) [reply src/renderer/components/reader/ReaderOptions.tsx]: Any thoughts about `predefined`? Isn't this completely unused now? (I could not find any references using text search in the Thorium codebase)

Comment (Scarsniik) [reply src/renderer/components/reader/ReaderOptions.tsx]: I already removed it

Comment (danielweck) [reply src/renderer/components/reader/ReaderOptions.tsx]: My apologies. Because of the Git "forced pushes" I missed the commit. Thank you! :)

**Pull request merged**